### DISCUSSION TITLE
How to backtest a long/short strategy?

### DISCUSSION THREAD

**Discussion by pedropivotto** [kernc/backtesting.py#601] [Q&A]
How to backtest a long/short strategy?

**Comment by kernc** [comment]
Have you seen the [tutorials](https://kernc.github.io/backtesting.py/doc/backtesting/#manuals)?

**Reply by atakanokan** [reply]
I think @pedropivotto 's question was valid to be honest. There is no tutorial on backtesting a L/S strategy (going long AND short on multiple tickers at the same time). One thing that I couldn't figure out was to feed in several tickers' OHLCV bars for example.

**Reply by atakanokan** [reply]
@kernc

**Reply by kernc** [reply]
@atakanokan Quick Start User Guide features this paragraphs:
> The library doesn't really support stock picking or trading strategies that rely on arbitrage or multi-asset portfolio rebalancing; instead, it works with an individual tradable asset at a time ...

Therefore, it looks like this is not yet available, unfortunately.

**Reply by atakanokan** [reply]
@kernc thank you for the prompt response. Is there a way to submit a recommendation for this on the feature map for the project?

Also, would it be possible to circumvent the issue by:

1. Assuming 10 tickers, create signals for all tickers and add them to the OHLCV dataframe
2. Loop through each of the tickers with different Backtest() instances
3. Go long or short if the ticker's signal warrants an action
4. Aggregate equity curves from 10 backtest

**Reply by BradKML** [reply]
Still thinking that someone else might have patched this https://github.com/kernc/backtesting.py/issues/1105#issuecomment-2016904405
